kyber_g1.go

```diff
@@ -10,33 +11,48 @@ import (
 	bls12381 "github.com/kilic/bls12-381"
 )
 
+// domainG1 is the DST used for hash to curve on G1, this is the default from the RFC.
+var domainG1 = []byte("BLS_SIG_BLS12381G1_XMD:SHA-256_SSWU_RO_NUL_")
+
+func DefaultDomainG1() []byte {
+	return domainG1
+}
+
 // KyberG1 is a kyber.Point holding a G1 point on BLS12-381 curve
 type KyberG1 struct {
 	p *bls12381.PointG1
+	// domain separation tag. We treat a 0 len dst as the default value as per the RFC "Tags MUST have nonzero length"
```

Comment:
I'm not sure I understand this comment

Reply:
If the dst is nil, the hash function will default to the default value.
Otherwise it uses it.